src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -77,8 +77,13 @@ public class URLClassLoader extends SecureClassLoader implements Closeable {
      * the URL is assumed to refer to a JAR file which will be downloaded and
      * opened as needed.
      *
+     * @apiNote If the {@code parent} is specified as {@code null} (for the
+     * bootstrap class loader) then there is no guarantee that all platform
+     * classes are visible.
```

Comment:
Should we add a link to `{@linkplain ClassLoader##builtinLoaders ...}` somewhere  in this API note? 

> there is no guarantee that all platform classes are visible

sounds (to me) like the behaviour is undefined (which is not the case). A link to the place where the various class loaders are discussed might clarify that.

Reply:
> Should we use this?

I missed Alan's suggestion to add a separate sentence linking to the ClassLoader section. I will update this PR accordingly.

Reply:
> something like "See Run-time Built-in Class Loaders for information on the system class loader other and the other built-in class loaders". I think that might work.

Done. Updated the PR with this text.

Reply:
No problem. Although I had seen that discrepancy, I assumed that you probably wanted to nudge users towards the use of system class loader for these constructors. I will replace it to bootstrap class loader shortly.